bin/compile

```diff
@@ -65,12 +82,21 @@ then
     ver=$(<$build/Godeps/Godeps.json jq -r .GoVersion)
 elif test -f $build/.godir
 then
+    warn "WARNING: .godir is deprecated and will be removed."
+    warn "Please use godeps or a canonical import path. For instructions:"
+    echo_urls
     name=$(cat $build/.godir)
-    ver=go${GOVERSION:-1.4.2}
+    ver=go${GOVERSION:-$DEFAULT_GOVERSION}
+elif go list -f {{.ImportComment}}
```

Comment:
I'm not sure this test does what you want. The exit status of this 'go list' command can be 0 even when there's no import comment. (And if there were an import comment, wouldn't this print out the value to the user?)

Maybe `elif test ! -z "$(go list -f {{.ImportComment}})"`?